src/Tools/dotnet-trace/TraceFileFormatConverter.cs

```diff
@@ -35,7 +35,15 @@ public static void ConvertToFormat(TraceFileFormat format, string fileToConvert,
                 case TraceFileFormat.NetTrace:
                     break;
                 case TraceFileFormat.Speedscope:
-                    ConvertToSpeedscope(fileToConvert, outputFilename);
+                    try
+                    {
+                        ConvertToSpeedscope(fileToConvert, outputFilename);
+                    }
+                    catch (Exception)
```

Comment:
Is there a specific exception type that is being thrown by TraceEvent?  If other exceptions get thrown we might not want to attempt to continue.